utils/Constants.ts

```diff
@@ -157,6 +157,11 @@ const initalBbstableForMinting = {
   }
 };
 
+export const blockTimeSeconds: { [key: string]: number } = {
+  rsk: 30,
+  mainnetFork: 14
+};
```

Comment:
Please also add already `arbitrumTestnet` with the value: 13

```suggestion
export const blockTimeSeconds: { [key: string]: number } = {
  rsk: 30,
  mainnetFork: 14,
  arbitrumTestnet: 13
};
```